hybrid/flow.py

```diff
@@ -163,6 +172,104 @@ def halt(self):
             component.stop()
 
 
+class Branches(Runnable, traits.MIMO):
+    """Parallelly executed :class:`~hybrid.core.Runnable` components.
+
+    Args:
+        *branches ([:class:`~hybrid.core.Runnable`]):
+            Runnable branches listed as positional arguments.
```

Comment:
Capitalization is inconsistent. Here "branches" but above "Branches". I'd go with branches in all descriptions except references like :class:`~hybrid.core.Runnable` because future updates will likely do that (no one reads it all before updating locally) and then where does it all end?

Reply:
branches -> plural of noun branch
Branches -> runnable component composed of multiple branches, each branch being of Branch type

That's my reasoning, is it incorrectly applied somewhere?